collection-scripts/gather_network_logs

```diff
@@ -2,141 +2,211 @@
 BASE_COLLECTION_PATH="must-gather"
 NETWORK_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/network_logs"}
 
-mkdir -p ${NETWORK_LOG_PATH}/
+mkdir -p "${NETWORK_LOG_PATH}"/
+
+function gather_multus_data { 
+  echo "INFO: Gathering Multus data"
+  #Should we store this on a per namespace basis? 
+  oc get net-attach-def -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/net_attach_def 2>&1 & PIDS=($!)
+
+  oc describe ippools.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/ippools  2>&1 & PIDS=($!)
+
+  oc describe overlappingrangeipreservations.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/overlappingrangeipreservations  2>&1 & PIDS=($!)
+}
+
+#This might be better situated in default must-gather scripts 
+function gather_namespace_specific_data { 
```

Comment:
This code isn't in master due to scale issues and should not be backported

Reply:
Instead these objects were added to the CNO's "related objects" list to be gathered by the `oc adm inspect` command